### ISSUE TITLE
subscriptions

### ISSUE THREAD

**Issue by stevezhuravel** [DataMascara/cisc3140-su19-project#88]
When you to subscriptions is it suppose to show what you are subscribed to if yes it does not show

**Comment by chalshaff12** [comment]
I don't think it is supposed to show your subscriptions when you add a new one. If that were the case, you would have to go back to the main page to subscribe to another port.. The way it is now, you can subscribe to many ports all at once. 

You can see your descriptions on your user Dashboard.
